src/lib/rnp.cpp

```diff
@@ -4058,6 +4058,8 @@ try {
     if (!key || !key->ffi || !output) {
         return RNP_ERROR_NULL_POINTER;
     }
+    bool need_armor = flags & RNP_KEY_EXPORT_ARMORED;
+    flags &= !RNP_KEY_EXPORT_ARMORED;
```

Comment:
We should not use logical not here (!) and use bitwise not (~) instead.

Reply:
Oops, my bad. Thanks for pointing out.